[CHORE] - Update setup files based on recent interactions

ju-Skinner opened the pull request:
# Description

Based on the recent use of the repo as we started, it was clear we needed to clean up a few things. This PR cleans up 

* Some formatting issues that were overlooked.
* Stencils Dev Server was loading the incorrect compiled dependencies
* Adds the `yarn.lock` to the repo
* Adds the `package.e2e.json` file to the `.gitignore`
* Removes the `format:check` from `lint_test_build` workflow.

cjbuchmann commented on app/assets/stylesheets/sage_docs/_code.scss:
```diff
@@ -10,21 +10,21 @@ pre.prettyprint {
+pre .atn { color: sage-color(primary, 200); font-weight: bold } /* attribute name  - khaki */
 pre .atv { color: #ffffff } /* attribute value - pink */
-pre .dec { color: sage-color(primary, 300) } /* decimal         - lightgreen */
+pre .dec { color: sage-color(primary, 200) } /* decimal         - lightgreen */
```
I think maybe these classes should be documented too? Maybe under styles?

ju-Skinner committed:
style(formatting): fix formatting from prettier

ju-Skinner committed:
fix: change the source directory for the Stencil devServer

When running e2e test it was not using the _generated directory to source the compiled files.

Change the outputPath for e2e to step back to folders, see comment in file

ju-Skinner committed:
chore(setup): add yarn.lock to repo

ju-Skinner committed:
fix: directory for proxy file was incorrect

This was creating a folder in the parent folder for the repo. We don't
want that.

ju-Skinner committed:
chore: add package.e2e.json to gitgnore

This file is generated when running e2e test.

ju-Skinner committed:
ci: attempt to resolve the 'No such ref: main' error

ju-Skinner committed:
ci: remove the format:check

The check does not give adequate info when an error has occurred
